src/api/pricer/CoinBase.ts

```diff
@@ -14,15 +14,14 @@ export type CoinBaseResponse = {
 
 const CURRENCY_MAPPING: CurrencyMapping = {
   RIF: 'RIF',
-  RBTC: 'RBTC',
   TRIF: 'RIF',
 };
 
 export default class CoinBase extends BaseExchangeApi {
   private readonly _httpWrapper: HttpWrapper;
 
   constructor(httpWrapper: HttpWrapper = new HttpWrapper()) {
-    super('CoinBase', CURRENCY_MAPPING, ['RIF', 'RBTC', 'tRif']);
+    super('CoinBase', CURRENCY_MAPPING, ['RIF', 'tRif']);
```

Comment:
why did we remove it from here?  should we able to recover from the error and test the next api?

Reply:
Yes, you're right, but I just wanted to fix the issue ASAP. I'll create a task to implement some sort of selection strategy, since there are some APIs (coinbase and coingecko) that support the same token (RIF)

Reply:
[Task](https://rsklabs.atlassian.net/browse/PP-755) created